vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/LanguageDescriptionObject.java

```diff
@@ -0,0 +1,53 @@
+package edu.asu.diging.vspace.core.model.impl;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.ILanguageDescriptionObject;
+import edu.asu.diging.vspace.core.model.IExhibitionLanguage;
+
+
+@Entity
+public class LanguageDescriptionObject implements ILanguageDescriptionObject {
```

Comment:
this name doesn't tell anything about what these objects are for (in fact it suggests that these objects describe languages, which they don't). How about `LocalizedText`?